src/views/DataDownloadView.vue

```diff
@@ -0,0 +1,191 @@
+          );
+          if (response.status === 200) {
+            return response.text();
+          } else {
+            return Promise.reject(
+              new Error("Unable to download data. Please try again later.")
+            );
+          }
+        })
+        .then((responseText) => {
+          this.isDataLoading = false;
+          // get csv file name from response Content-Disposition header
+          const regexMatch = responseContentDisposition.match(
+            /attachment; filename="([^"]+)"/
+          ); // Ex responseContentDisposition value: attachment; filename="some_file_name.csv"
+          let csvFileName = "data.csv"; //default csv file name
+          if (regexMatch && regexMatch.length > 0 && regexMatch[1]) {
+            csvFileName = regexMatch[1];
+          }
+          // download csv data to file
+          const csvDataText = responseText;
+          const anchor = document.createElement("a");
+          anchor.href =
+            "data:text/csv;charset=utf-8," + encodeURIComponent(csvDataText);
+          anchor.target = "_blank";
+          anchor.download = csvFileName;
+          anchor.click();
+        })
+        .catch((error) => {
+          this.isDataLoading = false;
+          this.dataDownloadErrorMsg = error.message;
+          console.log(error);
+        });
+    },
+  },
+};
+</script>
+<template>
+  <div class="container px-5 py-3" style="min-height: 60vh">
+    <h2 class="pb-2">Download G2P data by panel</h2>
+    <div
+      class="d-flex justify-content-center"
+      v-if="isDataLoading"
+      style="margin-top: 250px; margin-bottom: 250px"
+    >
+      <div class="spinner-border text-secondary" role="status">
+        <span class="visually-hidden">Loading...</span>
+      </div>
+    </div>
+    <div class="alert alert-danger mt-3" role="alert" v-if="panelErrorMsg">
+      <div>
+        <i class="bi bi-exclamation-circle-fill"></i> {{ panelErrorMsg }}
+      </div>
+    </div>
+    <div
+      v-if="
+        !isDataLoading &&
+        panelData &&
+        panelData.results &&
+        panelData.results.length > 0
+      "
+    >
+      <ul>
+        <li v-for="item in panelData.results">
+          <button
+            @click="downloadPanelData(item.name)"
+            type="button"
+            class="btn btn-link"
+            style="margin: 0; padding: 0; text-decoration: none"
+          >
+            {{ item.description ? item.description : item.name }} gene-disease
+            pairs and attributes
+          </button>
+        </li>
+      </ul>
+    </div>
+    <div
+      class="alert alert-danger mt-3"
+      role="alert"
+      v-if="dataDownloadErrorMsg"
+    >
+      <div>
+        <i class="bi bi-exclamation-circle-fill"></i> {{ dataDownloadErrorMsg }}
+      </div>
+    </div>
+    <div v-if="!isDataLoading">
+      <p>
+        Download format is available
+        <a href="" target="_blank" style="text-decoration: none">here</a>.
```

Comment:
We should have a README file under www.ebi.ac.uk/gene2phenotype/README (localhost:5173/README) and point to that.
I can create the file and send it to you.

Reply:
Plan is to host the README file in the FTP site and add a link to it.
I have to add a few files (this README file and other files related to the variant filtering documentation page) on the FTP site, once I've done that, I will raise a separate PR to update the links.